app/views/pages/item-detail/detail-mixin.js

```diff
@@ -0,0 +1,234 @@
+import _ from 'lodash';
+import moment from 'moment';
+import helpers from '../../components/helpers';
+import Gravatar from '../../components/gravatar';
+import OwnerAvatar from '../../components/item-card/owner';
+import Select from 'react-select';
+import {MentionsInput, Mention} from '@sprintly/react-mentions';
+import ProductActions from '../../../actions/product-actions';
+import ScoreMap from '../../../lib/score-map';
+import classNames from "classnames";
+import STATUS_MAP from '../../../lib/statuses-map';
+
+const selectPlaceholderMap = {
+  'assigned_to': "Choose assignee"
+}
+
+var DetailMixin = {
+  header(title) {
+    var titleCased = helpers.toTitleCase(title);
+
+    return (
+      <div className="header">
```

Comment:
I still think these mixin methods that return JSX would probably be better off as components.

We could even keep them all in one file since they're only a couple of lines each, but I'm concerned that we're missing out on potential virtual DOM rendering optimizations by not using "true" components.